swagger.json

```diff
@@ -4074,6 +4238,25 @@
                 "nl"
             ]
         },
+        "Title": {
+            "properties": {
+                "title": {
+                    "type": "string",
+                    "description": "The title of an item",
+                    "example": "Het Depot"
+                }
+            }
+        },
+        "Website": {
```

Comment:
In kader van bovenstaande opmerking ("url" ipv "website") zou ik dan voorstellen om dit te hernoemen naar "OrganizerUrl", omdat "Url" misschien te vaag is. Alhoewel als het echt herbruikbaar is kan dat misschien wel.